amd64/src/semantic.rs

```diff
@@ -747,7 +755,7 @@ pub fn call(a: Rvalue) -> Result<(Vec<Statement>, JumpSpec)> {
         call ?, new_rip:64;
     });*/
     let stmts = rreil!{
-        call ?, (a);
```

Comment:
The benefit of a second pass would also decouple a future decompiler backend from the IL used, as long as their was a trans from the IL -> IR, the decompiler could have multiple frontends :sunglasses: